HealthVault/Classes/Models/Generated/MHVActionPlan.m

```diff
@@ -88,5 +88,38 @@ + (NSDictionary *)objectParametersMap
     });
     return types;
 }
+@end
+@implementation MHVActionPlanCategoryEnum
+
+-(NSDictionary *)enumMap
+{
+    return @{
+        @"Unknown": @"Unknown",
+        @"Health": @"Health",
+        @"Sleep": @"Sleep",
+        @"Activity": @"Activity",
+        @"Stress": @"Stress",
+    };
+}
 
++(MHVActionPlanCategoryEnum *) Unknown
```

Comment:
I believe if you have an enum map, you don't need to do these initializer methods. You will have to silence the compiler warnings, but that will hopefully be pretty simple.